### PR TITLE
ImageProcessing 

### PR DESCRIPTION
**Pull request by yousufkanan**
Wrote an imaging processing script. Hard coded image resizing to 128x128 (if we want to change it we can). I decided to make it a square and if the user upload a non square image they will be prompted with a warning. Wrote tests that should work. i decided to force all images to be saved as a

### CODE REVIEW

File: Image_process/processImage.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        self.image_path = image_path
+
+    def resize_image(self):
+        """
+        Resize the image to the specified width and height.
+        :param width: Target width of the image.
+        :param height: Target height of the image.
+        :return: Resized Image object.
+        """
+        width = 128
```
Comment (jpappel) [inline Image_process/processImage.py:15]: width and height should be parameters of the method, setting the default values to 128x128 makes sense, so something like

```python
def resize_image(self, height: int = 128, width: int = 128):
    if height <= 0 or width <= 0:
        raise ValueError("height or width less than zero")
```

File: Image_process/processImage.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        #make sure file type is a jpg if not make it a jpg
+        if not save_path.endswith(".jpg"):
+            #remove the file extension before adding .jpg
+            save_path = save_path.split(".")[0]
+            save_path = save_path + ".jpg"
+        try:
+            image.save(save_path)
+        except Exception as e:
+            raise OSError(f"Failed to save image at {save_path}: {str(e)}") from e
+if __name__ == "__main__":
```
Comment (jpappel) [inline Image_process/processImage.py:39]: this does not belong here, it should be placed with the other tests

File: Database/ScanDynamo.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
+import json
```
Comment (jpappel) [inline Database/ScanDynamo.py:1]: this does not belong in this pr, you should be creating branches to avoid this issue

File: Image_process/processImage.py [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+        return image.resize((width, height))
+    
+    def save_image(self, image, save_path):
+        """
+        Save the image to the specified path.
+        :param image: Image object to save.
+        :param save_path: File path to save the image.
+        """
+        #make sure file type is a jpg if not make it a jpg
+        if not save_path.endswith(".jpg"):
```
Comment (jpappel) [inline Image_process/processImage.py:31]: should not be hardcoded with `jpg`, what if we wanted to support `png`'s or `gif`'s?

File: Image_process/processImage.py [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+        width = 128
+        height = 128
+        #if image isn't square raise a warning and resize to square
+        image = Image.open(self.image_path)
+        if image.size[0] != image.size[1]:
+            print("Warning: Image is not square. Resizing to square.")
+
+        return image.resize((width, height))
+    
+    def save_image(self, image, save_path):
```
Comment (jpappel) [inline Image_process/processImage.py:24]: assuming `save_path` to be a string will lead to issues when implementing saving files to s3. the second parameter of this function should be a path-like (`str`, `pathlib.Path`) or file-like (`io.BytesIO`) object

File: Image_process/processImage.py (hunk 4 above)
Comment (yousufkanan) [reply Image_process/processImage.py:31]: I haven't though of gif's I feel like stationary images should be a jpg because we don't need to focus too much on high quality images for the scope of this project

File: tests/test_lambda.py
Comment (jpappel) [inline tests/test_lambda.py:1]: this file should be renamed, it does not test lambda's (likely a separate pr which incorporates `localstack`)

File: Image_process/processImage.py [hunk 6]
```diff
@@ -0,0 +1,42 @@
+from PIL import Image
+
+
+class ImageProcess:
+    def __init__(self, image_path):
```
Comment (jpappel) [inline Image_process/processImage.py:5]: similar to my other comment, reading from only a string path will lead to issues with s3. also using an image path requires that we save an incoming upload to a file, then process it. if possible, this class should accept a stream or in-memory file-like object
